tools/harness-simulation/harness/Thread_Harness/THCI/OpenThread_Sim.py

```diff
@@ -37,28 +37,33 @@
 import paramiko
 import socket
 import time
+import win32api
 
 from IThci import IThci
 from OpenThread import OpenThreadTHCI, watched
 from simulation.config import REMOTE_OT_PATH
 
 
 class SSHHandle(object):
+    KEEPALIVE_INTERVAL = 30
 
-    def __init__(self, ip, port, username, password, device, node_id):
+    def __init__(self, ip, port, username, password, device, node_id, timeout):
         ipaddress.ip_address(ip)
         self.ip = ip
         self.port = int(port)
         self.username = username
         self.password = password
+        self.timeout = timeout
         self.__handle = None
         self.__stdin = None
         self.__stdout = None
         self.__connect(device, node_id)
+        win32api.SetConsoleCtrlHandler(self.__disconnect, True)
```

Comment:
What's the benefit of using this API?

Reply:
Close the SSH connection only when Harness exits. `win32api` has to be used because the library `signal` and/or `atexit` will not work when closing a window by pressing the X button on the upper right corner on Windows.

Disconnecting the SSH when Harness calls the corresponding method actually is not the intended behavior, because disconnecting and then connecting will automatically factory reset all states, while real devices remain the same state as before. As a result, once the SSH connection is established, it will be closed only when Harness exits.

Reply:
Should the connections be terminated automatically when Harness quit?

Reply:
Yes. It is an intended behavior. If we do not close the SSH connection explicitly, it will be closed as well when Harness quits, and then the simulations will be terminated after receiving the signal SIGHUP. Therefore, I think it is better to use the command `exit` to terminate the simulations automatically before Harness quits.

Reply:
make sense to me.